package com.prac.exception;

public class TryCatch {
    public static void main(String[] args) {
        //try{
            System.out.println("start try");
           // System.out.println(args[0]);
            System.out.println("end try");
       /* }catch (Exception e ){
            System.out.println("inside catch");
        }*/
        System.out.println("outside");
    }
}
